### ISSUE TITLE
Default DB

### ISSUE THREAD

**Issue by OlafvdSpek** [phpmyadmin/phpmyadmin#12102]
I'd like to be able to set a default DB in the .conf such that it's opened automatically when I open pMA.

**Comment by devenbansod** [comment]
I assume that the explicit URL parameters as explained in [FAQ 1.34]  (http://docs.phpmyadmin.net/en/latest/faq.html#can-i-access-directly-to-database-or-table-pages) should not be overridden by this. 
Any second opinions ?

**Comment by ibennetch** [comment]
I agree Deven, the precedence should be: explicit URL parameter is highest priority, then the `config.inc.php` default database setting from this issue, then finally `only_db` as fixed for #12158 as lowest priority.

**Comment by williamdes** [comment]
Seems to be implemented in 5.0.0

**Comment by OlafvdSpek** [comment]
How?

**Comment by williamdes** [comment]
#12158 ?

**Comment by OlafvdSpek** [comment]
See https://github.com/phpmyadmin/phpmyadmin/issues/12102#issuecomment-207676505
only_db is different from this.
